readme/acsf-setup.md

```diff
@@ -3,9 +3,11 @@
 To configure a project to run on ACSF, perform the following steps after initially setting up BLT:
 
 1. Execute `blt acsf:init` from the project root.
-1. Create a custom module in `docroot/modules/custom` that will be used in lieu of a custom profile.
-1. Add `acsf` as a dependency for your custom module.
-1. If using lightning, create a custom [lightning.extend.yml](https://github.com/acquia/lightning/blob/8.x-1.x/lightning.extend.yml) and add it to `sites/default`
+1. Create a custom subprofile as described [here](https://docs.acquia.com/lightning/subprofile).
+1. Add `acsf` as a dependency for your custom subprofile.
+1. Edit `blt/project.yml` to use the custom profile.
 1. Deploy to Cloud using `blt deploy`
 1. Use ACSF's "update code" feature to push the artifact out to sites.
-1. When creating a new site, select "Lightning" as the profile.
+1. When creating a new site, select your custom subprofile as the profile.
+
+**Note:** Code can be deployed via cloud hooks and the above process will not work for updating existing ACSF sites using lightning.
```

Comment:
Was trying to capture [@lcatlett's comment](https://github.com/acquia/blt/issues/1218#issuecomment-287906145). I guess I phrased it poorly. Not sure how you'd like to include that information